pkg/skaffold/kubernetes/logger/log.go

```diff
@@ -70,6 +76,15 @@ func NewLogAggregator(cli *kubectl.CLI, podSelector kubernetes.PodSelector, name
 		events:      make(chan kubernetes.PodEvent),
 		namespaces:  namespaces,
 	}
+	a.formatter = func(p v1.Pod, c v1.ContainerStatus, isMuted func() bool) log.Formatter {
```

Comment:
I think if you just remove the `isMuted` param here you can close over the existing function with no issues.

```suggestion
	a.formatter = func(p v1.Pod, c v1.ContainerStatus) log.Formatter {
```

Reply:
yes, I deliberately keep it separate. https://github.com/GoogleContainerTools/skaffold/pull/6337#discussion_r681177577